events/event/forms.py

```diff
@@ -20,20 +21,35 @@ def validate(self, value):
 class EventCreateForm(forms.ModelForm):
     class Meta:
         model = Event
-        exclude = ['participants', 'start_date', 'end_date']
+        exclude = ['participants', 'start_date', 'end_date', 'owner']
     start_date = DateValidator()
     end_date = DateValidator()
 
+    def clean_title(self):
+        title = self.cleaned_data['title']
+        if len(title) > 20:
+            raise forms.ValidationError("Cannot be more than 20 characters")
+        return title
+    
+    def clean_description(self):
+        description = self.cleaned_data['description']
+        if len(description) > 200:
+            raise forms.ValidationError("Cannot be more than 200 characters")
+        return description
+
     def clean_location(self):
-        data = self.cleaned_data.get('location')
+        location = self.cleaned_data['location']
         try:
-            isinstance(data[0], float) and isinstance(data[1], float)
+            isinstance(location[0], float) and isinstance(location[1], float)
         except:
```

Comment:
This is too general exception. What do you expect here? isinstance() returns only True or False. Exception can be raised here only if location is empty list or had data type that do not provide possibility to get element by position index.